BaCa2/BaCa2/settings.py

```diff
@@ -201,6 +201,30 @@
 BACA_PASSWORD = 'tmp-baca-password'
 BROKER_PASSWORD = 'tmp-broker-password'
 
+
+class BrokerRetryPolicy:
+    """Broker retry policy settings"""
+    # (In seconds) specify how many retries and how often should an
+    # HTTP post request be sent to the broker for one submit
+    individual_submit_retry_interval = 0.05
+    individual_max_retries = 5
+
+    # (In seconds) how long it should take for a submit to become expired
+    expiration_timeout = 180.0
+    # (In seconds) how many times a submit should be resent after it expires
+    resend_max_retries = 2
+    # (In minutes) how often should expiration check be performed
+    retry_check_interval = 60.0
+
+    # (In minutes) specify how old should error submits be before they are deleted
+    deletion_timeout = 60.0 * 24
+    # (In minutes) specify how often should the deletion check be performed
+    deletion_check_interval = 60.0 * 6
+
+    # Auto start broker daemons
+    auto_start = True
```

Comment:
takie rzeczy jak autostart pozostaw dla działania systemu - ogarniam je ja po stronie serwisów systemu ubuntu server. Muszą działać niezależnie od BaCy i podnosić się (docelowo) na osobnym obrazku dockera

Reply:
Tylko to jest część django i z tego co rozumiem trzeba to zastartować 'ręcznie' (w sensie trzeba wywołać metodę start na obiekcie scheduler) dla tego umieściłem wywołanie start w broker_api/__init__.py i dodałem opcję, żeby się to dało wyłączyć gdyby coś poszło nie tak.

Reply:
Trochę mylący ten komentarz napisałem bo ta zmienna nie dotyczy brokera samego w sobie, ale deamonów, które są odpowiedzialne za sprzątanie tabeli BrokerSubmit ze starych submitów, po stronie Bacy.

Reply:
ok